Makefile

```diff
@@ -2030,7 +2030,7 @@ openstack_kuttl_run: ## runs kuttl tests for the openstack operator, assumes tha
 	    oc delete osctlplane --all --namespace ${NAMESPACE}; \
 		make crc_storage_cleanup_with_retries; \
 		make crc_storage_with_retries; \
-		kubectl-kuttl test --config ${OPENSTACK_KUTTL_CONF} ${OPENSTACK_KUTTL_DIR} --test $${test_dir}; \
+		kubectl-kuttl test --config ${OPENSTACK_KUTTL_CONF} ${OPENSTACK_KUTTL_DIR} --test $${test_dir}; \ $(KUTTL_ARGS)
```

Comment:
`${KUTTL_ARGS}` is coming after a semi-colon on this line.

Reply:
good catch, i just did sed at all places :)

Reply:
Not adding at all to this target as already using --test arg in it.

Reply:
@bshephar can you check again